src/functionalTest/resources/features/F-011/F-011.feature

```diff
@@ -0,0 +1,34 @@
+@F-011
+Feature: F-011 : Verify CRD and JRD service bus messages
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-211
+  @FeatureToggle(EV:AZURE_SERVICE_BUS_FTA_ENABLED=on)
+  Scenario: must successfully verify CRD service bus messages for Legal Caseworker
```

Comment:
Should this not say:
```Scenario: must successfully create org role mapping for Legal Caseworker and Senior Legal Caseworker```

Reply:
Updated.

Reply:
Resolved in 4e0f7f55cae6678e4005fd17ca6d9e24538e691d